pkg/utils/utils.go

```diff
@@ -13,6 +13,10 @@ const (
 	// PXIncrementalCountAnnotation is the annotation used to set cloud backup incremental count
 	// for volume
 	PXIncrementalCountAnnotation = "portworx.io/cloudsnap-incremental-count"
+	// StorageClassAnnotationKey - Annotation key for storageClass
+	StorageClassAnnotationKey = "volume.beta.kubernetes.io/storage-class"
+	// StorageProvisionerAnnotationKey - Annotation key for storage provisioner
+	StorageProvisionerAnnotationKey = "volume.beta.kubernetes.io/storage-provisioner"
```

Comment:
You should also check for `volume.kubernetes.io/storage-provisioner` and use the definitions directly from here: https://github.com/kubernetes/kubernetes/blob/4516c7972d306918b8848488255d492f94d15aac/staging/src/k8s.io/component-helpers/storage/volume/pv_helpers.go#L74